models/User.js

```diff
@@ -1,7 +1,9 @@
-const { required } = require("joi");
+
+// const { required } = require("joi");
 const mongoose = require("mongoose");
 const bcrypt = require("bcryptjs");
-const jwt = require("jsonwebtoken");
+// const jwt = require("jsonwebtoken");
```

Comment:
You’ll need to remove the references to this in the User schema as well - it’s not throwing errors cause that methods no longer called but good practice